lib/ccnet/consts.js

```diff
@@ -0,0 +1,93 @@
+const SYNC = 0x02
+const ADDRESS = 0x03
+
+const commands = {
```

Comment:
Can make these all buffers by running the values through ``Buffer.from`` in lodash.